vnext/templates/cpp-app/windows/MyApp/MyApp.cpp

```diff
@@ -20,147 +20,61 @@ struct CompReactPackageProvider
+        .BundleRootPath(appDirectory)
 #if BUNDLE
-  host.InstanceSettings().JavaScriptBundleFile(L"index.windows");
-  host.InstanceSettings().BundleRootPath(std::wstring(L"file://").append(appDirectory).append(L"\\Bundle\\").c_str());
-  host.InstanceSettings().UseFastRefresh(false);
+        .UseFastRefresh(false)
 #else
-  host.InstanceSettings().JavaScriptBundleFile(L"index");
-  host.InstanceSettings().UseFastRefresh(true);
+        .UseFastRefresh(true)
 #endif
-
 #if _DEBUG
-  host.InstanceSettings().UseDirectDebugger(true);
-  host.InstanceSettings().UseDeveloperSupport(true);
+        .UseDirectDebugger(true)
+        .UseDeveloperSupport(true)
 #else
-  host.InstanceSettings().UseDirectDebugger(false);
-  host.InstanceSettings().UseDeveloperSupport(false);
+        .UseDirectDebugger(false)
+        .UseDeveloperSupport(false)
 #endif
+  };
 
-  winrt::Microsoft::ReactNative::ReactCoreInjection::SetTopLevelWindowId(
-      host.InstanceSettings().Properties(), reinterpret_cast<uint64_t>(hwnd));
-
-  winrt::Microsoft::ReactNative::Composition::CompositionUIService::SetCompositor(
-      host.InstanceSettings(), compositor);
+  auto packageProviders{winrt::single_threaded_vector<winrt::Microsoft::ReactNative::IReactPackageProvider>()};
 
-  return host;
-}
-
-_Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR /* commandLine */, int showCmd) {
-  // Initialize WinRT.
-  winrt::init_apartment(winrt::apartment_type::single_threaded);
-
-  // Enable per monitor DPI scaling
-  SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
-
-  // Create a DispatcherQueue for this thread.  This is needed for Composition, Content, and
-  // Input APIs.
-  auto dispatcherQueueController{winrt::Microsoft::UI::Dispatching::DispatcherQueueController::CreateOnCurrentThread()};
-
-  // Create a Compositor for all Content on this thread.
-  auto compositor{winrt::Microsoft::UI::Composition::Compositor()};
+  RegisterAutolinkedNativeModulePackages(packageProviders);
+  packageProviders.Append(winrt::make<CompReactPackageProvider>());
 
-  // Create a top-level window.
-  auto window = winrt::Microsoft::UI::Windowing::AppWindow::Create();
-  window.Title(windowTitle);
-  window.Resize({1000, 1000});
-  window.Show();
-  auto hwnd = winrt::Microsoft::UI::GetWindowFromWindowId(window.Id());
-  auto scaleFactor = ScaleFactor(hwnd);
+  winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
+  viewOptions.ComponentName(mainComponentName);
 
-  auto host = CreateReactNativeHost(hwnd, compositor);
+  // Initialize and Manage the ReactNativeHost
+  auto reactNativeAppBuilder{winrt::Microsoft::ReactNative::ReactNativeAppBuilder()
+                                 .AddPackageProviders(packageProviders)
+                                 .SetReactInstanceSettings(reactInstanceSettingsBuilder.ReactInstanceSettings())
+                                 .SetAppWindow(window)
```

Comment:
As per the above, don't set the window here.

Reply:
done.